distribution/lib/Standard/Table/0.0.0-dev/src/Extensions/Table_Ref.enso

```diff
@@ -1,6 +1,7 @@
 from Standard.Base import all
 import Standard.Base.Errors.Common.Index_Out_Of_Bounds
 import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+from Standard.Base.Data.Vector.Extensions import all
```

Comment:
These extensions shouldn't be exported from `Standard.Base` they should be `private` for each project. Otherwise they become part of API (which is what I'd like to avoid). `map_no_wrap` shalln't be visible to Enso users.

Reply:
They've been removed entirely.